glossary.md

```diff
@@ -14,15 +11,11 @@ This page contains a selection of key terms important to understanding the struc
 
 - **[Statistical variable](https://datacommons.org/browser/StatisticalVariable)**
 
-Any type of metric, statistic, or measure that can be measured at a place and time. Examples include median income, median income of females, number of high school graduates, unemployment rate, or prevalence of diabetes. For more information, see ["Intro to `StatisticalVariable` and `StatVarObservation`"](/contributing/background/representing_statistics.html).
+Any type of metric, statistic, or measure that can be measured at a place and time. Examples include median income, median income of females, number of high school graduates, unemployment rate, or prevalence of diabetes.
 
-- **[Statistical population](https://datacommons.org/browser/StatisticalPopulation)**
+- **[Statistical variable observation](https://datacommons.org/browser/StatVarObservation)**
 
-A set of instances of a certain given type that satisfy some set of constraints. Inherited from [schema.org definition](https://schema.org/StatisticalPopulation).
-
-- **[Observation](https://datacommons.org/browser/Observation)**
-
-When instantiated, can be used to specify observations about an entity at a particular time. Inherited from [schema.org definition](https://schema.org/Observation).
+A measurement of a StatisticalVariable for a particular place and time.
```

Comment:
Could we provide an example here as well, or more description?

Reply:
Yes! I added an example of a StatVarObservation here.